mem0/llms/anthropic.py

```diff
@@ -0,0 +1,67 @@
+import subprocess
+import sys
+import os
+import json
+from typing import Dict, List, Optional
+
+try:
+    import anthropic
```

Comment:
should we add anthropic in the poetry?

Reply:
I don't think we should do it, as anthropic is not a mandatory requirement.